pkg/datapath/ipcache/listener.go

```diff
@@ -115,6 +116,16 @@ func (l *BPFListener) OnIPIdentityCacheChange(modType ipcache.CacheModification,
 				"value": value.String()}).
 				Warning("unable to update bpf map")
 		}
+
+		if value.SecurityIdentity > identity.ReservedIdentityUnmanaged.Uint32() {
+			err = ipsec.UpsertIPSecEndpoint(cidr, ciliumIP)
```

Comment:
As for location, early on in this series I had per endpoint keys so every endpoint could possibly use its own key. This was a bit much for a first iteration even if we eventually get there on a per service basis for example. So I've left it hooked into ipcache so that we get these events. Anyways I'm not sure if we have a "endpoint subnet" has come online event anywhere.

On daemon startup we can flush the state/policy. I'll add this code now. This seems to be inline with other configuration we setup with Cilium. In general we don't tear it down on cilium exit but on daemon bring up.